### ISSUE TITLE
Fingerprinting's `prepend:` breaks sourcemaps path

### ISSUE THREAD

**Issue by kemenaran** [ember-cli/ember-cli#3918]
Using fingerprinting `prepend:` option along sourcemaps generations breaks the sourcemaps path.

The path to the sourcefile inserted at the bottom of the fingerprinted Javascript file is no longer relative to the JS file itself.

Steps to reproduce :
1. `ember new myapp`
2. Use the following configuration in `Brocfile.js`:

```javascript
var app = new EmberApp({
  sourcemaps: {
    enabled: true
  },
  fingerprint: {
    enabled: true,
    prepend: 'http://assets.myapp.com/'
  }
});
```
3. `ember serve`
4. Open the console

At the bottom of `myapp.js`, the link to the sourcemap is incorrect:
```
//# sourceMappingURL=assets/myapp-ab4f16713cc65a1e8f6840c4280ec04a.map
```

It doesn't work, and browsers fail to load the sourcemap. It should be:
```
//# sourceMappingURL=myapp-ab4f16713cc65a1e8f6840c4280ec04a.map
```

Any idea of where this comes from or how I could work around this issue?

*EDIT: changed the example path to http://assets.myapp.com/.*

**Comment by tchak** [comment]
I would actually expect it to be `http://localhost:4200/assets/myapp-ab4f16713cc65a1e8f6840c4280ec04a.map`...

**Comment by kemenaran** [comment]
Seems to be fixed in ember-cli `0.2.7` : the URL pointing to the sourcemap is now absolute.
